UI/platform-windows.cpp

```diff
@@ -498,7 +498,58 @@ void TaskbarOverlaySetStatus(TaskbarOverlayStatus status)
 	taskBtn->setOverlayIcon(icon);
 }
 #else
-// Needs to be re-implemented for Qt6, perhaps natively without Qt
-void TaskbarOverlayInit() {}
-void TaskbarOverlaySetStatus(TaskbarOverlayStatus) {}
+
+HWND hwnd;
+void TaskbarOverlayInit()
+{
+	hwnd = (HWND)App()->GetMainWindow()->winId();
+}
+
+void TaskbarOverlaySetStatus(TaskbarOverlayStatus status)
+{
+	ITaskbarList4 *taskbarIcon;
+	auto hr = CoCreateInstance(CLSID_TaskbarList, NULL,
+				   CLSCTX_INPROC_SERVER,
+				   IID_PPV_ARGS(&taskbarIcon));
```

Comment:
Does the `taskbarIcon` COM object get correctly Released on shutdown? I think the assumption is that it will (eg. when user attempts to close OBS, outputs will be terminated first) but is that a guarantee?